### PR TITLE
Check for unassigned fixed parameters

### PR DESCRIPTION
**Pull request by perost**
- Check that fixed parameters have binding equations during flattening.
- Also check that constants don't have fixed = false.

### CODE REVIEW

File: testsuite/flattening/modelica/scodeinst/Makefile [hunk 1]
```diff
@@ -39,7 +39,9 @@ AssertInvalid4.mo \
 AssignConstant1.mo \
 AssignInput1.mo \
 AssignModel1.mo \
-AssignPackage1.mo \
+AssignPackage1.mo \equation
+  x =
+
```
Comment (phannebohm) [inline testsuite/flattening/modelica/scodeinst/Makefile:44]: Typo?

File: OMCompiler/Compiler/NFFrontEnd/NFFlatten.mo [hunk 2]
```diff
@@ -741,6 +749,58 @@ algorithm
...
+        {ComponentRef.toString(var.name)}, var.info);
+      fail();
+    end if;
+
+    // Constants also must have binding equations if they are used, but this is
+    // checked when evaluating them.
+
+  else
+    if fixed and Binding.isUnbound(binding) and
+         Binding.isUnbound(Variable.lookupTypeAttribute("start", var)) then
```
Comment (phannebohm) [inline OMCompiler/Compiler/NFFrontEnd/NFFlatten.mo:795]: Can you give a warning if it is unbound but start attribute does exist?

File: testsuite/flattening/modelica/scodeinst/Makefile (hunk 1 above)
Comment (perost) [reply testsuite/flattening/modelica/scodeinst/Makefile:44]: Yes, must have pasted something by accident.

File: OMCompiler/Compiler/NFFrontEnd/NFFlatten.mo (hunk 2 above)
Comment (perost) [reply OMCompiler/Compiler/NFFrontEnd/NFFlatten.mo]: Done

**Review by phannebohm — APPROVED** [review]

**Pull request merged**